docs/aks/README.md

```diff
@@ -351,65 +322,68 @@ helm repo update
 
 ```
 
-## Install Linkerd Service Mesh into the cluster
+### Install AAD Pod Identity for the application
 
-Download the Linkerd v2 CLI:
+Change directories to the `docs/aks` folder and make the `aad-podid.sh` script executable. Running this shell script will deploy AAD Pod Identity to your cluster and assign a Managed Identity.
```

Comment:
why isn't the shell script already executable?  we should run a chmod +x and checkin so that can be removed